mysite/mysite/settings.py

```diff
@@ -85,7 +85,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'NAME': os.path.join(BASE_DIR, os.getenv('DATABASE_PATH')),
```

Comment:
De ce ai schimbat asta aici dar in .env example nu ai inclus DATABASE_PATH ?

Reply:
Eu as zice ca nu e nevoie de asta aici si nici de datele legate de database in env example, oricum sqlite nu e o baza de date adevarata, o sa facem asta pe urma.